tests/integration.py

```diff
@@ -156,6 +156,7 @@ def makeWebserverRequest(config, path, headers={}):
 class Test:
     """ tests will be automatically collected from this container class """
     PASS = None
+    SKIP = "SKIP"
 
```

Comment:
would a comment help with this because I actually can't really immediately see why this is neccesary